usr/share/rear/conf/examples/RHEL7-PPC64LE-Mulitpath-PXE-GRUB.conf

```diff
@@ -0,0 +1,37 @@
+# Default is to create Relax-and-Recover rescue media as ISO image
+# set OUTPUT to change that
+# set BACKUP to activate an automated (backup and) restore of your data
+# Possible configuration values can be found in /usr/share/rear/conf/default.conf
+#
+# This file (local.conf) is intended for manual configuration. For configuration
+# through packages and other automated means we recommend creating a new
+# file named site.conf next to this file and to leave the local.conf as it is.
+# Our packages will never ship with a site.conf.
+
+AUTOEXCLUDE_MULTIPATH=n
+BOOT_OVER_SAN=y
+
+OUTPUT=PXE
+OUTPUT_PREFIX_PXE=$HOSTNAME
+
+PXE_CONFIG_GRUB_STYLE=y
+# On your TFTP server:
+# In your tftpboot dir execute the following command to create the netboot directory based on GRUB.
+# grub2-mknetdir --net-directory=/var/lib/tftpboot
+# config directory will be /var/lib/tftpboot/boot/grub2/powerpc-ieee1275
+# More information about setting up a PXE based on Grub on POWER : https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/chap-installation-server-setup.html#sect-network-boot-setup-ppc-grub2
+PXE_CONFIG_URL="nfs://XX.YY.ZZ.AA/var/lib/tftpboot/boot/grub2/powerpc-ieee1275"
+PXE_CONFIG_PREFIX=grub.cfg-
```

Comment:
The only Global variable I created is `PXE_CONFIG_GRUB_STYLE` (because it must be set by local.conf)
The other variables I use in this example existed before my pull request (for lagacy PXE).
I tried to reuse the same variables that already exist when they point to the same thing (PXE_CONFIG_URL, PXE_CONFIG_PREFIX etc ...)

Don't mix up PXE_CONFIG_PREFIX and pxe_link_prefix I created in `810_create_pxelinux_cfg.sh`